samples/net/udp/boards/thingy91_nrf9160_ns.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Enable uart1 for tracing. */
+&uart1 {
+	status = "okay";
+	current-speed = < 1000000 >;
+	hw-flow-control;
```

Comment:
This property should not be here (in line with what we have in `snippets/nrf91-modem-trace-uart` )

Reply:
This is copy paste from changes that @eivindj-nordic added to some of our samples. cc @eivindj-nordic

Reply:
This is only added to applications that should have modem tracing enabled by default. If not, the snippet should be used.